build.sbt

```diff
@@ -141,6 +131,21 @@ lazy val library = (project in file("."))
         exclude("org.glassfish.hk2.external", "javax.inject"),
       "org.log4s" %% "log4s" % log4sVersion
     ),
+    assemblyMergeStrategy := {
+      case PathList("META-INF", xs@_*) => MergeStrategy.discard
+      case _ => MergeStrategy.first
+    },
+    assemblyShadeRules := {
+      val shadePackage = "cognite.spark.v1.shaded"
+      Seq(
+        ShadeRule.rename("shapeless.**" -> s"$shadePackage.shapeless.@1").inAll,
```

Comment:
The old commit also had `ShadeRule.rename("com.google.protobuf.**" -> "repackaged.cognite_spark_datasource.com.google.protobuf.@1").inAll,`, is it not needed?

Reply:
I didn't add this since the issue we're having is with cats. Anyway I'll add this as well.